packages/components/src/components/modal/_modal.scss

```
@@ -375,6 +375,10 @@
     z-index: z('modal');
   }
 
+  .#{$prefix}--body--with-modal-open .#{$prefix}--overflow-menu-options {
+    z-index: z('modal');
+  }
```

Could we combine this with the above selector?

```suggestion
// z-index fix for #8693
  .#{$prefix}--body--with-modal-open .#{$prefix}--tooltip, 
  .#{$prefix}--body--with-modal-open .#{$prefix}--overflow-menu-options {
    z-index: z('modal');
  }
```

---

Yes, good idea. I'll combine it.